gzsitl/gzsitl_plugin.cc

```diff
@@ -384,7 +365,7 @@ math::Pose GZSitlPlugin::coord_gzlocal_to_mavlocal(math::Pose gzpose)
 }
 
 void GZSitlPlugin::set_global_pos_coord_system(
-    mavlink_global_position_int_t position)
+    global_pos_int position)
```

Comment:
I think it is, since representing the position in fixed-point int32 is not very intuitive. I myself had to solve a bug when using the api on gzsitl because I got confused with this representation. What we might choose to do is to represent it as doubles instead, but that would be different from the mavlink standard for that type of message, so I'm not very sure what's the best approach.